webapi/src/test/scala/org/knora/webapi/e2e/v2/ResourcesRouteV2E2ESpec.scala

```diff
@@ -956,6 +956,38 @@ class ResourcesRouteV2E2ESpec extends E2ESpec(ResourcesRouteV2E2ESpec.config) {
                 )
            }
         }
+
+        "erase a resource" in {
```

Comment:
Could you add a test attempting to erase a resource that is referred to to make sure the request is rejected?

Reply:
There's a test for that in `ResourcesResponderV2Spec`:

https://github.com/dhlab-basel/Knora/pull/1429/files#diff-b1ef5075da04ef708a48de2a6b007436R2062

Reply:
Great. Sorry, I missed this one.